src/main/java/com/khutircraftubackend/user/UserController.java

```diff
@@ -20,24 +21,36 @@ public class UserController {
 
     private final ConfirmationService confirmationService;
 
-    @Operation(summary = "Confirm user registration", description = "Confirm user registration with email and confirmation token.")
+    @Operation(summary = "Confirm user registration", description = "Confirm user registration with confirmation token.")
     @ApiResponses(value = {
             @ApiResponse(responseCode = "200", description = "User confirmed successfully"),
-            @ApiResponse(responseCode = "400", description = "Invalid confirmation details")
+            @ApiResponse(responseCode = "400", description = "Invalid confirmation details"),
+            @ApiResponse(responseCode = "403", description = "Forbidden - Access Denied")
     })
+    @SecurityRequirement(name = "BearerAuth")
     @PostMapping(value = "/confirm", produces = "application/json")
```

Comment:
minor comment. 
"application/json" normally comes from MediaType.APPLICATION_JSON_VALUE
you'd need to import `import org.springframework.http.MediaType;`
check `CategoryController` - it has examples.